app/src/main/java/com/x8bit/bitwarden/data/platform/repository/SettingsRepositoryImpl.kt

```diff
@@ -535,6 +535,20 @@ class SettingsRepositoryImpl(
         settingsDiskSource.storeUseHasLoggedInPreviously(userId)
     }
 
+    override fun getShowAutoFillSettingBadge(userId: String): Boolean =
+        settingsDiskSource.getShowAutoFillSettingBadge(userId)
+
+    override fun storeShowAutoFillSettingBadge(userId: String, showBadge: Boolean) {
+        settingsDiskSource.storeShowAutoFillSettingBadge(userId, showBadge)
+    }
+
+    override fun getShowUnlockSettingBadge(userId: String): Boolean =
+        settingsDiskSource.getShowUnlockSettingBadge(userId)
+
+    override fun storeShowUnlockSettingBadge(userId: String, showBadge: Boolean) {
+        settingsDiskSource.storeShowUnlockSettingBadge(userId, showBadge)
```

Comment:
Missing tests for this one.

Reply:
<img src="https://media3.giphy.com/media/xT5LMzIK1AdZJ4cYW4/giphy.gif"/>